src/components/Semester.vue

```diff
@@ -205,7 +205,7 @@ export default {
       this.isShadow = false;
       this.isDraggedFrom = false;
     });
-    init = false;
+    const init = false;
```

Comment:
this init is probably useless?

Reply:
Yes I don't really know the purpose of this init in Semester.vue. I will try removing it and see if anything breaks.